content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -52,34 +52,33 @@ rules:
 
 Alpha metric →  Stable metric →  Deprecated metric →  Hidden metric → Deletion
 
-Alpha metrics have no stability guarantees; as such they can be modified or deleted at any time.
+Alpha metrics have no stability guarantees. These metrics can be modified or deleted at any time.
 
-Stable metrics can be guaranteed to not change; Specifically, stability means:
+Stable metrics are guaranteed to not change. This means:
+* A stable metric will not be deleted or renamed
```

Comment:
Is this true? We have sections on deprecation and deletion, are these only for Alpha metrics?

Reply:
To be more precisely, a stable metric `without a deprecated signature` will not be deleted or renamed.

Reply:
Thanks for the clarification!